frontend/src/metabase/actions/components/ActionForm/ActionFormFieldWidget.tsx

```diff
@@ -8,7 +8,6 @@ import FormRadioWidget, {
 import FormSelectWidget from "metabase/core/components/FormSelect";
 import FormNumericInputWidget from "metabase/core/components/FormNumericInput";
 import FormBooleanWidget from "metabase/core/components/FormToggle";
-import CategoryFieldPicker from "metabase/components/FormCategoryInput";
```

Comment:
we were using this component to pull in field values for implicit forms, but it didn't have a way to add values not reported by the API. In any event, the backend field metadata is now sparser, so we can't ever use this because we don't have a field's `semantic_type` and `field_id` in the current API data.